machine-learning/Luna-Forecasting.ipynb

```diff
@@ -0,0 +1,507 @@
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "// Configure AutoML\n",
```

Comment:
I would convert the inline code commets to Markdown cells and also split up the code into individual steps and add more details:

- Split train/test data
- Initialize monitor
- Create experiment

Reply:
Also, is there a reason we call `Result` directly on the async operation. We should use

`var res = await experiment.Run()`;

Reply:
Would that makes cell too small which causes a notebook having too many cells to run? A long cell might frighten users off, but too many cells might also use up one's patience..